cmd/vulcan-prowler/main.go

```diff
@@ -74,13 +96,59 @@ var (
 			"https://github.com/toniblyx/prowler",
 			"https://www.cisecurity.org/benchmark/amazon_web_services/",
 		},
+		Score: report.SeverityThresholdLow,
+	}
+
+	CISLevel2Compliance = report.Vulnerability{
+		Summary: "Compliance With CIS Level 2 AWS Foundations Benchmark (BETA)",
+		Description: `<p>
+			This account has been checked for compliance with the CIS Level 2 according
+			to its security classification. You can check the security classification
+			of the account in the details section.
+			</p>
+			<p>
+		    The CIS AWS Foundations Benchmark provides prescriptive
+			guidance for configuring security options for a subset of Amazon Web
+			Services with an emphasis on foundational, testable, and architecture
+			agnostic settings. The services included in the scope are: IAM, Confing, 
+			CloudTrail, CloudWatch, SNS, S3 and VPC (Default).
+		</p>
+		<p>
+			The provided recommendations are provided as a result of controls included in
+			the CIS Level 2.
+		</p>
+		<p>
+			Check the Details and Resources sections to know the compliance status
+			and more details.
+		</p>`,
+		References: []string{
+			"https://d0.awsstatic.com/whitepapers/compliance/AWS_CIS_Foundations_Benchmark.pdf",
+			"https://github.com/toniblyx/prowler",
+			"https://www.cisecurity.org/benchmark/amazon_web_services/",
+		},
+		Score: report.SeverityThresholdMedium,
+	}
+
+	CISComplianceInfo = report.Vulnerability{
+		Summary: "Information about CIS AWS Foundations Benchmark (BETA)",
+		Description: `<p>
+			     Information gathered by executing the CIS benchmark about the account.
+		</p>
+			`,
+		References: []string{
+			"https://d0.awsstatic.com/whitepapers/compliance/AWS_CIS_Foundations_Benchmark.pdf",
+			"https://github.com/toniblyx/prowler",
+			"https://www.cisecurity.org/benchmark/amazon_web_services/",
+		},
+		Score: report.SeverityThresholdNone,
 	}
 )
 
 type options struct {
 	Region          string   `json:"region"`
 	Groups          []string `json:"groups"`
 	SessionDuration int      `json:"sessionDuration"` // In secs.
```

Comment:
I think we are doing `snake_case` for all the options in Vulcan. Maybe this is a good chance to change this or maybe we can do it in a separate PR.

Reply:
Yes makes sense, I'm doing it in this PR because, as you said, we are already using it to improve a little bit the check and not only for the taking into account the security level. I'm also modifying the local.toml.example file.